app/customer.py

```diff
@@ -0,0 +1,46 @@
+from math import dist
+from typing import List
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self, customer: dict) -> None:
+        self.name: str = customer["name"]
+        self.product_cart: dict = customer["product_cart"]
+        self.location: list = customer["location"]
+        self.money: int = customer["money"]
+        self.car: dict = customer["car"]
+
+    def fuel_cost(self, fuel_price: float, shop_location: list) -> float:
+        distance = dist(self.location, shop_location)
+        return self.car["fuel_consumption"] * distance / 100 * fuel_price
+
+    def product_cost(self, product_shop: dict) -> float:
+        return sum(product_shop[product] * cost
+                   for product, cost in self.product_cart.items())
+
+    def cheap_store(self, fuel_price: float, shops: List[Shop]) -> tuple:

```

Comment:
"cheap_store" ?
what is this function about?
rename it for better understanding what she does